core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -110,13 +110,16 @@ object ConsumerGroupCommand extends Logging {
   }
 
   def printOffsetsToReset(groupAssignmentsToReset: Map[String, Map[TopicPartition, OffsetAndMetadata]]): Unit = {
-    if (groupAssignmentsToReset.nonEmpty)
-      println("\n%-30s %-30s %-10s %-15s".format("GROUP", "TOPIC", "PARTITION", "NEW-OFFSET"))
+    val format = "%-30s %-30s %-10s %-15s"
+    if (groupAssignmentsToReset.nonEmpty) {
+      println()
```

Comment:
Why not move this `println()` to `format`？ So is the one below.

Reply:
Seems like explicit `println` makes it easier to understand what will be printer.
But, of course, we can go with more compact 
`println("\n" + format.format("GROUP", "TOPIC", "PARTITION", "NEW-OFFSET"))`

Reply:
> Why not move this println() to format？ So is the one below.

Done.